test/recipes/30-test_evp.t

```diff
@@ -14,15 +14,20 @@ use OpenSSL::Test qw(:DEFAULT data_file bldtop_dir);
 
 setup("test_evp");
 
-my @files = ( "evpciph.txt", "evpdigest.txt", "evpencod.txt", "evpkdf.txt",
-    "evppkey_kdf.txt", "evpmac.txt", "evppbe.txt", "evppkey.txt",
-    "evppkey_ecc.txt", "evpcase.txt", "evpaessiv.txt", "evpccmcavs.txt" );
+my @configs = qw( default.cnf legacy.cnf default-and-legacy.cnf fips.cnf );
```

Comment:
Perhaps for this test it is sufficient just to run `default-and-legacy.cnf` and `fips.cnf`?

Reply:
Sure...